th2_common/schema/message/impl/rabbitmq/abstract_rabbit_subscriber.py

```diff
@@ -103,14 +105,21 @@ def handle(self, channel, method, properties, body):
                         listener.handler(self.attributes, value)
                     except Exception as e:
                         logger.warning(f"Message listener from class '{type(listener)}' threw exception {e}")
+        except DecodeError as e:
+            logger.exception(f"Can not parse value from delivery for: {method.consumer_tag} due to DecodeError: {e}\n"
```

Comment:
Please, use single quotes with f strings, unless single quote should be included in string

Reply:
No problem. It will be done but what's the idea to use single quotes?

Reply:
In Python, single-quoted strings and double-quoted strings are the same. PEP 8 does not make a recommendation for this. However, most style guides recommend using single quotes by default, unless you need single quote character in your string.